### DISCUSSION TITLE
2024.3.1

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#3997] [Announcements]
This release has minor fixes, like fix for the desktop app crashing on Windows 10, locale and i18n updates, and fix for a sandbox escape vulnerability for `@hoppscotch/js-sandbox` (fix introduced in release `0.8.0` of the CLI).

## What's Changed
* fix: use proper values for `addTo` field when auth type is `api-key` by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3966
* update locale hu by @krisztianbarat in https://github.com/hoppscotch/hoppscotch/pull/3875
* fix: spanish lang translation messages by @sawa-ko in https://github.com/hoppscotch/hoppscotch/pull/3950
* fix(desktop): set window caption color if Windows version >= 11 by @Timic3 in https://github.com/hoppscotch/hoppscotch/pull/3939
* i18n(fr): Updating the `packages/hoppscotch-common/locales/fr.json` file. by @thomasbnt in https://github.com/hoppscotch/hoppscotch/pull/3555
* refactor: infra-config code refactor by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/3982
* chore: migrate `Node.js` implementation for `js-sandbox` to `isolated-vm` by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/3973
* fix: french lang translation by @jobartim44 in https://github.com/hoppscotch/hoppscotch/pull/3986
* fix: use pressed key rather than its code by @posva in https://github.com/hoppscotch/hoppscotch/pull/3978
* feat: first time user spotlight animation by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3977

## New Contributors
* @krisztianbarat made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3875
* @Timic3 made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3939
* @jobartim44 made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3986
* @posva made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3978

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2024.3.0...2024.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2024.3.1'>2024.3.1</a>.</em>

**Comment by tinycocha** [comment]
when can you support disabled ssl cert verified on desktop app?

**Reply by AndrewBastin** [reply]
We have it and general Desktop App stability as a task for May.
